README.md

```diff
@@ -1,22 +1,96 @@
 
-moja global welcomes a wide range of contributions as explained in [Contributing document](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md) and in the [About moja-global Wiki](https://github.com/moja-global/.github/wiki).  
+## Usage
 
-  
-## FAQ and Other Questions  
+To set up this project locally follows the below procedure:
 
-* You can find FAQs on the [Wiki](https://github.com/moja.global/.github/wiki).  
-* If you have a question about the code, submit [user feedback](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Provide-User-Feedback.md) in the relevant repository  
-* If you have a general question about a project or repository or moja global, [join moja global](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Join-moja-global.md) and 
-    * [submit a discussion](https://help.github.com/en/articles/about-team-discussions) to the project, repository or moja global [team](https://github.com/orgs/moja-global/teams)
-    * [submit a message](https://get.slack.help/hc/en-us/categories/200111606#send-messages) to the relevant channel on [moja global's Slack workspace](mojaglobal.slack.com). 
-* If you have other questions, please write to info@moja.global   
-  
+1. Clone [FLINT.Example](https://github.com/moja-global/FLINT.Example) repository. A `Dockerfile` is inside the `Docker` folder. So to build the image follow the below command:
+
+```shell
+cd Docker
+docker build --build-arg NUM_CPU=8 -t moja/flint.example:bionic .
+```
+
+It will take some time to build the image. For more information, you can follow [https://docs.moja.global/en/latest/DevelopmentSetup/docker_installation_example.html](https://docs.moja.global/en/latest/DevelopmentSetup/docker_installation_example.html).
+
+2. Clone [FLINT.UI](https://github.com/moja-global/FLINT-UI) repository:
+
+When using submodules the installation code needs to be:
+
+```shell
+git clone --recursive https://github.com/moja-global/flint-ui
+```
+
+Or if you've already initialized the repository without the submodule
+
+```shell
+git submodule update --init --recursive
+```
+
+Now you have to get inside the repository directory and to build the `docker-compose.yml` file follow the below command:
+
+```shell
+docker-compose up
+```
+
+This will build all the docker images inside the `docker-compose.yml` file. It will take some time and after that, you can see all images list in the Docker app.
+
+<div align="center">
+<img src="assets/docker-images.jpg">
+</div>
+
+`docker-compose.yml` file is compose of three services i.e. `flint.example.api`, `flint.gcbm.api` and `flint.ui`.
+
+Now you can run any all the containers by clicking on the `RUN` button. You can use the command line too to run these images. All containers list will look like this in the Docker app:
+
+<div align="center">
+<img src="assets/docker-containers.jpg">
+</div>
+
+<br />
+
```

Comment:
Can you please the instruction: "To view the UI please navigate to localhost:{{default port in docker-compose.yml}}" <- but with the correct default port?

Reply:
A screenshot of the UI would be great too!